src/static/app/login/login.html

```diff
@@ -1,6 +1,33 @@
-   <!-- Remind Passowrd -->
-    <base href="/">
-    <div id="formFooter p-3">
-      <button onclick="location.href='/signup'" class="blue-btn underlineHover" type="button">Sign Up</button>
+<section class="login">
+  <div class="wrapper fadeInDown">
```

Comment:
This format is correct right? Is it possible to have less background layers? Also could the sign up button be a different color? Like our red, blue or white? If another color is needed, let me know and we can add to our current brand colors. (see top of css files for hex values of our red and blue, feel free to make the red/blue darker or lighter if needed but keep it similar). 

One last thing, the font and button need to have good color contrast, if background is dark, the font needs to be light or the background light, and the dark font. 

<img width="650" alt="Screen Shot 2023-04-24 at 5 34 35 PM" src="https://user-images.githubusercontent.com/17969555/234123266-47653267-9355-46d2-88fc-bdee32a1835f.png">